bin/init.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 Google. All rights reserved. Use of this source code is
+// governed by a BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+/**
+ * A command to create a simple, starting `tool/grind.dart` build script.
+ */
+void main(List args) {
+  File file = new File('tool${Platform.pathSeparator}grind.dart');
```

Comment:
I think this should either validate that we are at the root of a pub package (`pubspec.yaml` exists) or try to climb the file system until a `pubspec.yaml` is found and produce a path relative to that.